src/accordion.tsx

```diff
@@ -194,19 +199,22 @@ const AccordionItemContent = React.forwardRef<TouchableElement, AccordionItemCon
 );
 
 export const AccordionItem = React.forwardRef<TouchableElement, AccordionItemContentProps>(
-    ({dataAttributes, ...props}, ref) => (
-        <AccordionItemContent
-            {...props}
-            ref={ref}
-            dataAttributes={{'component-name': 'AccordionItem', ...dataAttributes}}
-        />
+    ({dataAttributes, role = 'listitem', ...props}, ref) => (
```

Comment:
Adding list/listitem roles by default (similar to Rows)

Reply:
This solves the relationship between the accordion group and each of the accordions in case they have relation between them, since they could also be only regions of content with no relation between them.

How the relationship between the header and the content can be defined?

I see two cases:

* They are a group of controls: The accordion item acts as a `fieldset` where the header title is the legend and the controls are inside the collapsable panel.
* They are a definition list (usually used for FAQs): The list of accordions acs as a definition list so the accordion header should have the role: "term" and the content of the collapsable section, the role: "definition"

Reply:
I'm not sure I understood you. Currently, the relation between the header and the content is made by using aria-controls in the header and aria-labelledby + role=region in the panel.

Is that what you meant?

Reply:
Discussed offline. Alex will investigate about best practices regarding this and will come back to us with some updates. In case some changes are necessary, we can iterate on it in a different PR.